charts/gxf/templates/deployment.yaml

```diff
@@ -36,34 +36,26 @@ spec:
         image: {{ .Values.image }}{{- if .Values.imageTag -}}:{{- end -}}{{ .Values.imageTag }}
         resources: {{- toYaml .Values.resources | nindent 10 }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
-
         env:
-          - name: NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          - name: APP_NAME
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.labels['app']
-          #The environment variables below
-          #are used directly by the application
-          #they are the environment variable versions
-          #of the variables defined in *.properties files.
-          #'db.username' = DB_USERNAME
-          - name: DB_USERNAME
-            value: osp_admin
```

Comment:
DB_USERNAME environment variable is no longer needed here?

Reply:
I suspect this is unnecessary/redundant as in every properties file the value is already set to osp_admin:
- https://github.com/OSGP/open-smart-grid-platform/search?l=Java+Properties&p=3&q=db.username
- https://github.com/SmartSocietyServices/osgp-applications/search?l=Java+Properties&p=2&q=db.username